cmd/helm/root.go

```diff
@@ -134,31 +135,44 @@ func newRootCmd(actionConfig *action.Configuration, out io.Writer, args []string
 	flags.ParseErrorsWhitelist.UnknownFlags = true
 	flags.Parse(args)
 
+	commandGroups := templates.CommandGroups{
+		{
+			Message: "Release Commands:",
```

Comment:
I would re-word this as "Release Management Commands:". That way it's more descriptive about when you should use these commands.

Reply:
👍，updated